packages/visualizations/addon/templates/components/navi-visualizations/table.hbs

```diff
@@ -14,6 +14,9 @@
                 value=(readonly column.displayName)
                 focus-out=(action 'updateHeaderDisplayName' column)
               }}
+              {{#if (or (eq column.type 'metric') (eq column.type 'threshold'))}}
+                {{number-format-dropdown column=column onUpdateReport=onUpdateReport}}
```

Comment:
`classNames=table-widge__number-format-dropdown`

Reply:
Since this feature isn't ready, do we have a feature flag somewhere to hide this)?

Reply:
The feature flag has been added in https://github.com/yahoo/navi/pull/91/files#diff-2b01e32ee456dad575a2128ca8e9e300R48